src/V3EmitCFunc.h

```diff
@@ -35,19 +35,19 @@ constexpr int EMITC_NUM_CONSTW = 8;
 // Emit lazy forward declarations
 
 class EmitCLazyDecls final : public VNVisitor {
-    // NODE STATE/TYPES
-    //  AstNode::user2() -> bool. Already emitted decl for symbols.
-    const VNUser2InUse m_inuser2;
 
     // MEMBERS
     std::unordered_set<string> m_emittedManually;  // Set of names already declared manually.
     EmitCBaseVisitor& m_emitter;  // For access to file output
     bool m_needsBlankLine = false;  // Emit blank line if any declarations were emitted (cosmetic)
+    std::unordered_set<AstNode*> m_emitted;  // -> in set. Already emitted decl for symbols.
```

Comment:
```suggestion
    std::set<AstNode*> m_emitted;  // -> in set. Already emitted decl for symbols.
```

As unordered_set clearing time will otherwise likely dominate.

Reply:
Fixed.